genie_annotations.json

```diff
@@ -0,0 +1,24 @@
+{
+  "tissueSource": [
+    "Blood",
+    "Solid Tumor"
+  ],
+  "center": [
+    "NKI",
+    "DFCI",
+    "GRCC",
+    "JHU",
+    "MSK",
+    "UHN",
+    "VICC",
+    "MDA"
+  ],
+  "platform": [
```

Comment:
HiSeq2000 is consistent with platform names in the neurannotations dictionary that we have already accepted. We can discuss standardization of these names at a future meeting, but I think for now it is okay to accept since it is consistent across projects.